recipes/ambit/meta.yaml

```diff
@@ -0,0 +1,143 @@
+        - if not exist %PREFIX%\\Library\\share\\cmake\\ambit\\ambitConfig.cmake exit 1  # [win]
+        - if not exist %PREFIX%\\Library\\share\\cmake\\ambit\\ambitTargets-static.cmake exit 1  # [win]
+        # Inspect linkage
+        - ldd -v $PREFIX/lib/libambit$SHLIB_EXT                                # [linux]
+        - otool -L $PREFIX/lib/libambit$SHLIB_EXT                              # [osx]
+        - objdump.exe -p %PREFIX%\\Library\\lib\\ambit.lib | findstr /i "dll"  # [win]
+
+  - name: pyambit
+    script: build-py.sh                                                        # [unix]
+    script: bld-py.bat                                                         # [win]
+    build:
+      skip: true                                                               # [win]
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - cmake
+        - ninja
+      host:
+        - {{ pin_subpackage('ambit', exact=True) }}
+        - python
+        - pybind11
+        - pybind11-abi
+        - numpy
+        - libblas                                                              # [win]
+        - liblapack                                                            # [win]
+        - intel-openmp                                                         # [win]
+      run:
+        - {{ pin_subpackage('ambit', exact=True) }}
+        - python
+        - intel-openmp                                                         # [win]
+    test:
+      requires:
+        - pip                                                                            # provides `pip check`
+        - pytest
+        - numpy
+        - m2w64-binutils                                                       # [win]   # provides objdump.exe
+      files:
+        - test_ambit.py
+      imports:
+        - ambit
+      commands:
+        - ls -l $SP_DIR/ambit                                                  # [unix]
+        - dir %SP_DIR%\\ambit                                                  # [win]
+        # Verify module
+        - test -f $SP_DIR/ambit/pyambit*.so                                    # [unix]
+        - if not exist %SP_DIR%\\ambit\\pyambit*.pyd exit 1                    # [win]
+        # Verify accessories
+        - test -f $PREFIX/share/cmake/ambit/ambitConfig.cmake                  # [unix]
+        - test -f $PREFIX/share/cmake/ambit/ambitTargets-Python.cmake          # [unix]
+        - if not exist %PREFIX%\\Library\\share\\cmake\\ambit\\ambitConfig.cmake exit 1  # [win]
+        - if not exist %PREFIX%\\Library\\share\\cmake\\ambit\\ambitTargets-Python.cmake exit 1  # [win]
+        # Inspect linkage
+        - ldd -v $SP_DIR/ambit/pyambit*.so                                     # [linux]
+        - otool -L $SP_DIR/ambit/pyambit*.so                                   # [osx]
+        - objdump.exe -p %SP_DIR%\\ambit\\pyambit.*.pyd | findstr /i "dll"     # [win]
```

Comment:
For my curosity, what additional information are you gaining by inspecting linkages here? Conda-build already reports linkage information from the binaries in your package.

Reply:
Mostly habit and familiarity of output. The `ldd` gives the full paths and transitive linkage and glibc info that the conda output doesn't. The `objdump` line has been helpful when placed in `bld.bat` to provide some openmp clues _before_ the conda-build dep analysis fails or tests fail. For the final working recipe, the linkage info seemed best to live in tests.

In pre-c-f, pre-conda-build-3 incarnations of these recipes using local build resources, I used to fake a sysroot and then carefully check the glibc versions and library locations. So except for the windows use case above, inspecting linkages is mostly paranoia at this point.